Update troubleshooting.md (#1838)
pulumi/pulumi-hugo: themes/default/content/docs/support/troubleshooting.md

when absolutely needed.
{{% /notes %}}

```bash
$ pulumi up --logtostderr --logflow -v=9 2> out.txt
```

> Resource providers may also have their own flags or environment variables that control diagnostic logging.  For example, for any Pulumi providers that use a bridged Terraform provider can make use of the [`TF_LOG`](https://www.terraform.io/docs/internals/debugging.html) environment variable (set to `TRACE`, `DEBUG`, `INFO`, `WARN` or `ERROR`) in order to provider additional diagnostic information.

Reviewer: ```suggestion
Diagnostic logging can also be controlled with flags and environment variables of the resource providers. For example, Pulumi providers that use a bridged Terraform provider can make use of the [`TF_LOG`](https://www.terraform.io/docs/internals/debugging.html) environment variable (set to `TRACE`, `DEBUG`, `INFO`, `WARN` or `ERROR`) in order to provide additional diagnostic information.
```

are there other situations than with the bridged providers? if not, suggest being even more specific and clear than im suggesting here.

Author: I think your description here is more accurate. Accepting.
